joshuaCarpentier/assignments/treesBFS/lc111/lc111.js

```diff
@@ -11,10 +11,59 @@ class TreeNode {
   }
 };
 
+/**
+ * 
+ * Input: tree
+ * Output: the maximum depth level
+ * 
+ * Variables: root, root. left, root.right, index, array to store the root 
+ * 
+ * EdgeCase different condition for different inputs
+ * 
+ * Brute force Approach
+ * --------------- 
+ * Pre-Condition: check if empty 
+ * 
+ * Conditions: there is only really a pre condition since were only looking for the shortest path to root
+ * 
+ * Termination-Conditions: check if left and right are null
 
+ * 
+ * Post-Conditions: 
```

Comment:
For an approach, it would be better if you detailed what you code was going to do and how in sentences. By looking at this approach, I am not sure what your solution is going to do or how it will solve the prompt.